database/scripts/0002-populate-data.sql

```diff
@@ -21,7 +21,16 @@ INSERT INTO public.event_rating(would_recommend, created_by, event_id) VALUES
     (false, 2, 2),
     (true, 2, 5);
 
+INSERT INTO public.user_settings(id, receive_top_rated, user_id) VALUES
+    (1, true, 1),
+    (2, false, 2);
+
+INSERT INTO public.sport_subscriptions(id, user_id, sport_id) VALUES
+    (1, 1, 1),
+    (2, 1, 3);
+
 -- Postgres requires us to reset autoincrement counters manually if we're inserting data by explicitly setting IDs
 SELECT setval('users_id_seq', (SELECT MAX(id) from public.users));
 SELECT setval('sports_id_seq', (SELECT MAX(id) from public.sports));
-SELECT setval('events_id_seq', (SELECT MAX(id) from public.events));
\ No newline at end of file
+SELECT setval('events_id_seq', (SELECT MAX(id) from public.events));
+SELECT setval('user_settings_id_seq', (SELECT MAX(id) from public.user_settings));
```

Comment:
Don't we need this for `sport_subscriptions` as well?

Reply:
We sure do :D
https://github.com/mbrizic/ratethegame/pull/28/commits/1209c70c384f25049c1b9abd5c0b2c4db815aeb1